modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -353,9 +354,16 @@ private ObjectDefinition _toObjectDefinition(
 						com.liferay.portal.kernel.util.PropsUtil.get(
 							"feature.flag.LPS-152650"))) {
 
-					restContextPath =
-						objectDefinition.isSystem() ? StringPool.BLANK :
-							("/o" + objectDefinition.getRESTContextPath());
+					if (StringUtil.startsWith(
+							objectDefinition.getRESTContextPath(), "/")) {
+
+						restContextPath =
+							"/o" + objectDefinition.getRESTContextPath();
+					}
+					else {
+						restContextPath =
+							"/o/" + objectDefinition.getRESTContextPath();
+					}
```

Comment:
````
restContextPath = StringBundler.concat(
objectDefinition.isSystem() ? "/o/" : "/o", objectDefinition.getRESTContextPath());
````

Reply:
formatsource don't let me do exactly this,
so I do:
`				restContextPath = objectDefinition.isSystem() ? "/o/" :
					"/o" + objectDefinition.getRESTContextPath();`